cmd/node/node.go

```diff
@@ -370,12 +374,19 @@ func (app *SpacemeshApp) Start(cmd *cobra.Command, args []string) {
 
 	app.unregisterOracle = func() { oracleClient.Unregister(true, pub.String()) }
 
-	bo := oracle.NewBlockOracleFromClient(oracleClient, int(app.Config.CONSENSUS.NodesPerLayer))
+	//bo := oracle.NewBlockOracleFromClient(oracleClient, int(app.Config.CONSENSUS.NodesPerLayer))
+	nodesPerLayer := app.Config.CONSENSUS.NodesPerLayer
+	layersPerEpoch := app.Config.CONSENSUS.LayersPerEpoch
+	activationDb := &activation.ActivationDb{}      // TODO: initialize properly
```

Comment:
There is an implementation of this in our code, however it is disconnected from all flows. I think we should first run tests on each component separately and then combine them together